admin/src/main/scala/io/buoyant/admin/AdminConfig.scala

```diff
@@ -2,4 +2,4 @@ package io.buoyant.admin
 
 import io.buoyant.config.types.Port
 
-case class AdminConfig(port: Port = Port(9990))
+case class AdminConfig(port: Port = Port(9990), shutdownGraceMs: Int = 10000)
```

Comment:
Sure, I was trying to emulate what you do with port. Any reason why you want to it differently in this case?